admin/workspace-providers.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Workspace Providers
+description: Learn how workspace providers can minimize latency for users.
+state: beta
+---
+
+Coder allows you to add to your deployment multiple Kubernetes clusters. These
+additional clusters are called **providers**. This can be helpful for
+distributed teams, since users can choose the provider closest to them to
+minimize latency.
+
+## Enabling Workspace Providers
+
+To enable the Workspace Providers UI elements:
+
+1. In the Coder UI, click on your avatar.
+2. In the menu that appears, click **Feature Preview**.
+3. Select **Resource Pools UI** and click **Enable**.
+
+Once enabled, this allows end-users to choose the workspace provider and
+namespace where they want their environment deployed.
+
+## Admin UI
+
+Site managers and admins can view the workspace providers configuruation
+page avaiable via **Manage** > **Admin** > **Workspace Providers**.
+
+![Workspace Providers Admin](../assets/workspace-providers-admin.png)
+
+The Admin panel will provide high-level information on all of your workspace
+providers, as well as whether the provider is active on not. The default tag
```

Comment:
on not -> or not (but I suggest omitting "or not", as the sentence reads fine without it)

I would suggest:

> The Admin panel shows an overview of all configured workspace providers, and indicates whether they are active. The "default" tag indicates the provider that is initially selected when a user creates an environment using the Create an Environment dialog.

(the word "modal" always felt kinda jargon-y to me, frontend engineers would know what it means but users might not care about the distinction between a modal dialog and other dialogs)